cmsplugin_articles_ai/__init__.py

```diff
@@ -2,7 +2,7 @@
 #      .--.
 #       /    \
 #      ## a  a       _
-#      (   '._)     |_|
+#      (   '._)    c|_|
```

Comment:
I'd also might remove this butler. Or maybe if we enforce these for all the plugins then this butler can live on